src/Microsoft.PowerShell.Commands.Utility/commands/utility/Var.cs

```diff
@@ -332,6 +369,23 @@ public string[] Exclude
                 ExcludeFilters = value;
             }
         }
+        /// <summary>
+        /// Specify the SessionState to get variable from.
+        /// </summary>
+        [Parameter]
+        public SessionState SessionStateEntry
+        {
+            get
+            {
+                return _sessionState;
+            }
+
+            set
+            {
+                _sessionState = value;
+            }
+        }
+        private SessionState _sessionState;
```

Comment:
Would it make more sense to declare this as an *inherited param* from VariableCommandBase? (In which case you'd stick the definition up around lines 24-30.)

Might simplify the definitions a bit. ^^

Reply:
Placing repetitious code into VariableCommandBase works well however the SessionStateEntry is a Parameter. Parameters should stay with its relative function so we don't confuse/obfuscate the code base. Also isn't documentation generated from its <Summary> Tags? Then the parameter would become UnDocumented in get-help which is not pretty.